build/MUX-Release.yml

```diff
@@ -0,0 +1,110 @@
+name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+jobs:
+- job: ComponentDetection
+  pool:
+    vmImage: 'VS2017-Win2016'
+
+  steps:
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+    displayName: 'Component Detection'
+
+- job: Build
+  pool:
+    name: Package ES Custom Demands Lab A
+    demands:
+      - ClientAlias -equals depcontrols2
+  timeoutInMinutes: 120
+  strategy:
+    maxParallel: 10
+    matrix:
+      Debug_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Debug'
```

Comment:
Do we need Debug configuration for a release?

Reply:
No, I suppose not. I'll try removing it.